MessagingLibrary/RabbitMq/RabbitMqPublisher.cs

```diff
@@ -11,8 +11,11 @@ public class RabbitMqPublisher : IMessagePublisher
     private readonly string _hostname;
     private readonly string _exchange;
     private readonly string _exchangeType;
+    private readonly int _port;
+    private readonly string _username;
+    private readonly string _password;
 
-    public RabbitMqPublisher(string hostname, string exchange, string exchangeType)
+    public RabbitMqPublisher(string hostname, string exchange, string exchangeType, int port, string username, string password)
```

Comment:
lets have the rabbitmq configs gathered in a special class and pass it here please.